_data/banking.yml

```
@@ -857,18 +1073,24 @@ websites:
       - sms
       - phone
     doc: https://www.mbna.ca/help-centre/credit-cards/two-step-verification-faq/
+    regions:
+      - ca
 
   - name: Meridian Credit Union
     url: https://www.meridiancu.ca
     img: meridian.png
     twitter: MeridianCU
     facebook: MeridianCreditUnion
+    regions:
+      - ca
 
   - name: Monzo
     url: https://www.monzo.com
     twitter: monzo
     facebook: monzobank
     img: monzobank.png
+    regions:
+      - gb
```

```suggestion
      - gb
      - us
```
Monzo now [services the US](https://monzo.com/usa/) and has [started opening accounts](https://monzo.com/blog/24/01/2020/us-product-roadmap). They've partnered with [Sutton Bank](https://www.suttonbank.com/) as the issuer, which means accounts are insured by FDIC. They use the [same login for US and UK](https://trello.com/c/0bAVlLP0/60-the-ability-to-switch-between-uk-and-us-contexts-within-the-app-if-you-have-both-accounts).